client/src/Backoffice/components/Main/components/Rides/Rides.tsx

```diff
@@ -64,7 +64,7 @@ const columns: ColumnDef<Partial<Ride>>[] = [
   },
   {
     accessorKey: 'comment',
-    header: 'הערות',
+    header: 'תיאור הנסיעה',
```

Comment:
@aviv-taori 
In the figma it looks like the text is "מטרת הנסיעה".

Reply:
@OrleeTal @LEVI-RIVKIN 
Orly's directive that all screens be compatible

Reply:
@aviv-taori 

Did orlee direct you to use the phrase "תיאור הנסיעה" instead of "מטרת הנסיעה" on all screens?
In the figma it seems like the phrase should be "מטרת הנסיעה" on all screens.

@OrleeTal Can you clarify?